src/data-table.component.ts

```diff
@@ -71,6 +71,10 @@ export class DataTableComponent implements AfterViewInit, OnInit, OnDestroy, OnC
     }
     return numRows;
   }
+  /**
+   * Native table indentifier.
+   */
+  @Input() public customId: string;
```

Comment:
if we propagate it further as id, the main should be id as well.
`<ic-data-table id="xyz"></ic-data-table>` or `<ic-data-table [id]="'xyz'"></ic-data-table>`